### DISCUSSION TITLE
SO V2.3.250, firewall group addition question

### DISCUSSION THREAD

**Discussion by StephenEHays** [Security-Onion-Solutions/securityonion#10506] [Unsupported Versions] [answered]
Just realize the SO's firewall is  blocking LAN access to my network controller software that is normally accessible. Using Ubuntu 20 with tplink omada network controller software. I attempted to modify /opt/so/saltstack/default/salt/firewall/portgroups.yaml adding a port group but had to revert back as SO stopped working, I think I need to define this group in another way. any advise? just looking to allow these ports as services are running.

have also attempted via so-firewall

**Comment by dougburks** [comment]
Are you saying you installed third party software directly on Security Onion? 

You may be able to make it work, but we don't recommend it or support it:
https://docs.securityonion.net/en/2.3/best-practices.html#avoid-third-party-software-and-modifications

**Reply by StephenEHays** [reply, marked as answer]
Not what i'm saying at all. The controller was installed on Ubuntu server then Security Onion was installed later. Not a problem though and do sweat it. If assistance is not available here, I will just figure this out on my own. After digging deeper it looks like I will need to script the iptables.

Docker and iptables try  ferm: https://github.com/MaxKellermann/ferm/

works great.
